.ci/Jenkinsfile

```diff
@@ -79,7 +119,7 @@ pipeline {
             def agents = readYaml(file: '.ci/.jenkins-agents.yml')
             def parallelTasks = [:]
             agents['agents'].each { agent ->
-              parallelTasks["apm-agent-${agent.NAME}"] = generateStepForAgent(repo: "${agent.REPO}", featuresPath: "${agent.FEATURES_PATH}")
+              parallelTasks["apm-agent-${agent.NAME}"] = generateStepForAgent(repo: "${agent.REPO}", filesType: "json", filesPath: "${agent.JSON_SPECS_PATH}")
```

Comment:
Skip agents that don’t define the JSON_SPECS_PATH yet

Reply:
Ahh good catch! my fault, I was preparing the code commit-by-commit to allow that and forgot it in the last commit, thanks for  👀 !